### PR TITLE
credrank: make parameters accessible on MPG

### PR DESCRIPTION
**Pull request by decentralion**
We need the parameters to be accessible on the object, so that we can
reference them when creating the fibration edges.

Test plan: I added a simple test. :)

### CODE REVIEW

File: src/core/credrank/markovProcessGraph.test.js [hunk 1]
```diff
@@ -400,6 +395,16 @@ describe("core/credrank/markovProcessGraph", () => {
       const edgesExpected = edgeOrder.map((x) => mpg.edge(x));
       expect(edgesActual).toEqual(edgesExpected);
     });
+    it("has the correct epoch boundaries for the given intervals", () => {
+      const expected = [-Infinity, 0, 2, Infinity];
+      expect(markovProcessGraph().epochBoundaries()).toEqual(expected);
+    });
+    it("has the right participants", () => {
+      expect(markovProcessGraph().participants()).toEqual([participant]);
+    });
```
Comment (wchargin) [inline src/core/credrank/markovProcessGraph.test.js:404]: Is this test related? Maybe meant for a different commit?

**Review by wchargin — APPROVED** [review]

File: src/core/credrank/markovProcessGraph.test.js (hunk 1 above)
Comment (decentralion) [reply src/core/credrank/markovProcessGraph.test.js:418]: I was just adding extra tests for consistency (testing all of the "trivial" property accessors)
